ga/23.0.0.6/kernel/Dockerfile.ubuntu.ibmjava8

```diff
@@ -95,20 +69,27 @@ RUN set -eux; \
     curl -LfsSo /usr/bin/dumb-init ${DUMB_INIT_URL}; \
     echo "${DUMB_INIT_SHA256} */usr/bin/dumb-init" | sha256sum -c -; \
     chmod +x /usr/bin/dumb-init; \
-    apt-get purge --auto-remove -y wget; \
+    apt-get purge --auto-remove -y curl; \
```

Comment:
Are you sure we want to remove curl here?  I thought it was in the base image, but perhaps not for the ibmjava8.  Is this related to the strange warnings we got about wget on Java8?

Reply:
I can delete that line (and the following one), but we do a 
```
apt-get update; \
apt-get install -y curl; \
```
in that block, and then there was a apt-get purge for wget, so I figured that was done in error and it was meant to be curl (since we don't install wget).

Reply:
right, the uninstall of wget is an issue since I doubt it is installed at this point.  I seem to recall curl is not in the ibmjava8 images.  If that is the case then uninstall here is appropriate.  I suspect curl is not in ibmjava8 because that is the only one we do an install of curl before downloading dumb-init.

Reply:
right, that was my guess as well based on it only being in the java8 image.  Hence the 'fix' of changing wget to curl.  So unless I'm misunderstanding, we're going to keep it as is.